core/global/platform.gd

```diff
@@ -48,6 +51,8 @@ func _init() -> void:
 	# Set hardware platform provider
 	if PLATFORM.ABERNIC_GEN1 in flags:
 		platform = load("res://core/platform/abernic_gen1.tres")
+	if PLATFORM.AOKZOE_GEN1 in flags:
+		platform = load("res://core/platform/aokzoe_gen1.gd")
```

Comment:
You should load the `.tres` file instead of the script directly.

Reply:
Good catch. Woops.